Harvest.Web/Controllers/Api/QuoteController.cs

```diff
@@ -39,6 +39,22 @@ public async Task<ActionResult> Get(int projectId)
             return Ok(model);
         }
 
+        // Get info on the project as well as in-progess quote if it exists
+        [HttpGet]
+        public async Task<ActionResult> GetApproved(int projectId)
+        {
+            var project = await _dbContext.Projects
+                .Include(p => p.Quote)
```

Comment:
@sprucely FYI, this is currently null. This PR will start populating it:
https://github.com/ucdavis/Harvest/pull/591

Reply:
Wasn't null in my tests against local db.